checkov/terraform/checks/resource/oci/AbsSecurityListUnrestrictedIngress.py

```diff
@@ -0,0 +1,40 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class AbsSecurityListUnrestrictedIngress(BaseResourceCheck):
+    def __init__(self, check_id, port):
+        name = f"Ensure VCN inbound security lists allow all traffic on {port} port."
+        supported_resources = ['oci_core_security_list']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=check_id, categories=categories, supported_resources=supported_resources)
+        self.port = port
+
+    def scan_resource_conf(self, conf):
+        if 'ingress_security_rules' in conf:
+            self.evaluated_keys = ['ingress_security_rules']
+            rules = conf.get("ingress_security_rules", [])
+
+            for idx, rule in enumerate(rules):
+                if "0.0.0.0/0" in rule['source'][0] \
+                        and not (
+                        (rule['protocol'][0] != '1' and ('udp_options' not in rule) and ('tcp_options' not in rule))
+                        or (self.scan_protocol_conf(rule, 'tcp_options', idx) != CheckResult.FAILED
+                            and self.scan_protocol_conf(rule, 'udp_options', idx) != CheckResult.FAILED)
+                        or rule['protocol'][0] == 'all'):
+                    self.evaluated_keys = [f'ingress_security_rules/[0]/[{idx}]']
+                    return CheckResult.FAILED
+
+            return CheckResult.PASSED
+
+        return CheckResult.FAILED
```

Comment:
@ayajbara I believe this is correct for SSH port, but not for RDP. Would add another parameter to the abs class, something like `is_exposed_by_default`, and set it to `True` for port 22 and `False` for port 3389
Then do something like this:
```suggestion
        return CheckResult.FAILED if self.is_exposed_by_default else CheckResult.PASSED
```